integrations/varnish/documentation.yaml

```diff
@@ -0,0 +1,143 @@
+exporter_type: sidecar
+app_name_short: Varnish
+app_name: {{app_name_short}}
+app_site_name: Varnish
+app_site_url: https://varnish-cache.org/
+exporter_name: the Varnish exporter
+exporter_pkg_name: prometheus_varnish_exporter
+exporter_repo_url: https://github.com/jonnenauha/prometheus_varnish_exporter
+dashboard_available: true
+minimum_exporter_version: "1.6.1"
+multiple_dashboards: false
+dashboard_display_name: {{app_name_short}} Prometheus Overview
+config_mods: |
+  apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    name: varnish
+    namespace: default
+  spec:
+    selector:
+      matchLabels:
+  +     app.kubernetes.io/name: varnish
+    template:
+      metadata:
+        name: varnish
+        labels:
+  +       app.kubernetes.io/name: varnish
+      spec:
+        containers:
+        - name: varnish
+          image: varnish:7.2.0
+  +       volumeMounts:
+  +         - name: shared-data
+  +           mountPath: /var/lib/varnish
```

Comment:
can we add
```
        env:
          - name: VARNISH_HTTP_PORT
            value: "8080"
```
to this sample yaml? Yes, we write these guides from the perspective that the user already has varnish up and running, but just need to edit the config to include the exporter, but if you remove all the "+"s we should still expect the resulting yaml to work out of the box. 

I tried this and got [this issue](https://github.com/varnish/docker-varnish/issues/42), but looks like setting the port fixes it.